release/src/version-helpers.ts

```diff
@@ -209,14 +207,16 @@ export function versionSort(a: string, b: string) {
 
 export function getLastReleaseFromTags({
   tags,
-  ignorePatches = false
+  ignorePatches = false,
+  ignorePreReleases = false,
```

Comment:
this function was always ignoring preReleases, which is the behavior we want when we're looking for the last minor release in order to build release notes, which this function is also used for. This introduces a flag so that we can find a different "last" commit based on context.